providers/env-var/src/main/java/dev/openfeature/contrib/providers/envvar/EnvironmentKeyTransformer.java

```diff
@@ -0,0 +1,113 @@
+package dev.openfeature.contrib.providers.envvar;
+
+import java.util.function.*;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.text.CaseUtils;
+
+/**
+ * This class provides a way to transform any given key to another value. This is helpful, if keys in the code have a
```

Comment:
would be nice if we add an info about this capability in readme.

Reply:
thanks for pointing it out. it actually forced me to change the implementation a little to avoid additional configuration when using key transformations